### DISCUSSION TITLE
Lower audio latency?

### DISCUSSION THREAD

**Discussion by E-daw** [wwmm/easyeffects#1881] [Q&A]
I just installed easy effects the other day and have configured it on my laptop to make the speakers sound MUCH MUCH better. But I notice all audio has an extreme amount of latency at 133ms (specified by easy effects at the bottom of the application.) Is there a way to mitigate this latency or lower it? Not necessarily to 0 but something significantly lower than 100ms? I'm having trouble finding guides or other people with the same complaint.

(On version 6.30 of easy effects, Manjaro linux on 5.15.71 kernel, installed from pacman)

Thanks in advance

**Comment by wwmm** [comment]
It depends on which plugins you are using and on how they are configured. Many of them do not even add latency. Are you using the equalizer in `FIR` mode or any other mode different from `IIR`? Depending on how the equalizer is configured it may add a lot of latency.

**Reply by E-daw** [reply]
I just adjusted the EQ into IRR mode and it reduced down to 5ms thank you a lot!! Is there anything I can do to lower it further? I'm using an EQ, a convolver and a limiter.

**Reply by wwmm** [reply]
I think that the equalizer in IIR mode does not add latency or at least not one that should be easy to measure with our current tools. These 5 ms may be coming from the limiter. Is it configured to use `lookahead`?
